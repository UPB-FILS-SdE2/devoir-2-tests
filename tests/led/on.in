led on 0 2
quit