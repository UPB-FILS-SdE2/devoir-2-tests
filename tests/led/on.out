cmd: Set pixel 0 2 to 9
cmd: 